catroid/src/main/java/org/catrobat/catroid/ui/adapter/BrickAdapter.java

```diff
@@ -1138,6 +1144,22 @@ private boolean brickHasAFormula(Brick brick) {
 		return (brick instanceof FormulaBrick || multiFormulaValid);
 	}
 
+	private boolean isBrickWithoutDescription(Brick brick) {
+		String name = brick.getClass().getSimpleName();
+		if (name.equals(IfLogicElseBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(IfLogicEndBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(IfThenLogicEndBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(LoopEndlessBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(LoopEndBrick.class.getSimpleName())) {
+			return true;
+		}
+		return false;
+	}
+
```

Comment:
Why do I compare the classes like that, good question. That is redundant. I was still in the mentality from the previous commit where I had to use the simple name for the URL.

I also agree with your suggestion according the rename of the function. I had not thought of that. 

If Thomas deems those changes worthy enough to un-merge the commit, I will be sure to adjust the code accordingly. Thanks!